documentation/commands/cmd-status.txt

```diff
@@ -25,8 +25,8 @@
 ~
 ~ --secret: 
```

Comment:
Assuming we're trying to introduce a different key for the purposes of getting status, should we change the flag from `--secret` to `--status-key` to avoid confusion?